lifetiles-graph/src/main/java/nl/tudelft/lifetiles/graph/controller/GraphController.java

```diff
@@ -107,10 +110,14 @@ public final void initialize(final URL location,
         listen(Message.OPENED,
                 (controller, args) -> {
                     assert controller instanceof MenuController;
-                    assert args[0] instanceof File && args[1] instanceof File;
+                    assert args[0] instanceof String;
+                    if (!((String) args[0]).equals("graph")) {
```

Comment:
I think it would be nicer to change the first `shout` method argument to a `String` as sender and giving each controller a unique ID (or use its class name). This will remove alot of casts.